### PR TITLE
Update TextMate2.munki.recipe

### PR DESCRIPTION
**Pull request by Daz-wallace**
As recommended by the vender. See issue #360
Links with PR #361

### CODE REVIEW

File: TextMate/TextMate2.munki.recipe [hunk 1]
```diff
@@ -21,10 +21,8 @@ preferences panel.
         <string>apps/TextMate2</string>
         <key>NAME</key>
         <string>TextMate2</string>
-        <key>BRANCH</key>
-        <string>release</string>
-        <key>MACOS</key>
-        <string>10.15</string>
+        <key>PRERELEASE</key>
+        <string></string>
```
Comment (nmcspadden) [inline TextMate/TextMate2.munki.recipe:25]: This isn't necessary to include in this recipe.

**Review by nmcspadden — CHANGES_REQUESTED** [review]
Please include the output of `autopkg run -vv` for this recipe.

File: TextMate/TextMate2.munki.recipe (hunk 1 above)
Comment (Daz-wallace) [reply TextMate/TextMate2.munki.recipe]: Sorted 😊 output incoming

**Comment by Daz-wallace** [comment]
```
% autopkg run -vvv /Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate/TextMate2.munki.recipe 
Processing /Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate/TextMate2.munki.recipe...
WARNING: /Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate/TextMate2.munki.recipe is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...
{'AUTOPKG_VERSION': '2.1',
 'GIT_PATH': '/Applications/Xcode.app/Contents/Developer/usr/bin/git',
 'MUNKI_REPO': '/Users/darren/munki_repo',
 'MUNKI_REPO_SUBDIR': 'apps/TextMate2',
 'NAME': 'TextMate2',
 'PARENT_RECIPES': ['/Users/darren/Library/AutoPkg/RecipeRepos/com.github.autopkg.recipes/TextMate/TextMate2.download.recipe'],
 'PRERELEASE': '',
 'RECIPE_CACHE_DIR': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2',
 'RECIPE_DIR': '/Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate',
 'RECIPE_OVERRIDE_DIRS': ['~/Library/AutoPkg/RecipeOverrides'],
 'RECIPE_PATH': '/Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate/TextMate2.munki.recipe',
 'RECIPE_REPOS': 
 **[TRUNCATED],**
 'RECIPE_SEARCH_DIRS': 
 **[TRUNCATED],**
 'pkginfo': {'catalogs': ['testing'],
             'description': "TextMate brings Apple's approach to operating "
                            'systems into the world of text editors. By '
                            'bridging UNIX underpinnings and GUI, TextMate '
                            'cherry-picks the best of both worlds to the '
                            'benefit of expert scripters and novice users '
                            'alike.',
             'display_name': 'TextMate 2',
             'minimum_os_version': '10.12',
             'name': 'TextMate2',
             'postinstall_script': '#!/bin/sh\n'
                                   '[ -d /usr/local/bin ] || mkdir -p '
                                   '/usr/local/bin\n'
                                   '/bin/cp -f -p '
                                   '/Applications/TextMate.app/Contents/Resources/mate '
                                   '/usr/local/bin/mate\n',
             'postuninstall_script': '#!/bin/sh\n'
                                     '[ -e /usr/local/bin/mate ] && rm -f '
                                     '/usr/local/bin/mate\n',
             'unattended_install': True},
 'verbose': 3}
GitHubReleasesInfoProvider
{'Input': {'github_repo': 'textmate/textmate', 'include_prereleases': ''}}
GitHubReleasesInfoProvider: No value supplied for CURL_PATH, setting default value of: /usr/bin/curl
GitHubReleasesInfoProvider: Selected asset 'TextMate_2.0.6.tbz' from release 'v2.0.6'
{'Output': {'url': 'https://github.com/textmate/textmate/releases/download/v2.0.6/TextMate_2.0.6.tbz',
            'version': '2.0.6'}}
URLDownloader
{'Input': {'filename': 'TextMate2-2.0.6.tbz',
           'url': 'https://github.com/textmate/textmate/releases/download/v2.0.6/TextMate_2.0.6.tbz'}}
URLDownloader: No value supplied for prefetch_filename, setting default value of: False
URLDownloader: No value supplied for CHECK_FILESIZE_ONLY, setting default value of: False
URLDownloader: Item at URL is unchanged.
URLDownloader: Using existing /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/downloads/TextMate2-2.0.6.tbz
{'Output': {'pathname': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/downloads/TextMate2-2.0.6.tbz'}}
EndOfCheckPhase
{'Input': {}}
{'Output': {}}
Unarchiver
{'Input': {'archive_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/downloads/TextMate2-2.0.6.tbz',
           'destination_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2',
           'purge_destination': True}}
Unarchiver: Guessed archive format 'tar_bzip2' from filename TextMate2-2.0.6.tbz
Unarchiver: Unarchived /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/downloads/TextMate2-2.0.6.tbz to /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2
{'Output': {}}
CodeSignatureVerifier
{'Input': {'input_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2/TextMate.app',
           'requirement': 'identifier "com.macromates.TextMate" and anchor '
                          'apple generic and certificate '
                          '1[field.1.2.840.113635.100.6.2.6] /* exists */ and '
                          'certificate leaf[field.1.2.840.113635.100.6.1.13] '
                          '/* exists */ and certificate leaf[subject.OU] = '
                          '"45TL96F76G"'}}
CodeSignatureVerifier: Verifying code signature...
CodeSignatureVerifier: Deep verification enabled...
CodeSignatureVerifier: Strict verification not defined. Using codesign defaults...
CodeSignatureVerifier: /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2/TextMate.app: valid on disk
CodeSignatureVerifier: /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2/TextMate.app: satisfies its Designated Requirement
CodeSignatureVerifier: /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2/TextMate.app: explicit requirement satisfied
CodeSignatureVerifier: Signature is valid
{'Output': {}}
DmgCreator
{'Input': {'dmg_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2.dmg',
           'dmg_root': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2'}}
DmgCreator: Created dmg from /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2 at /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2.dmg
{'Output': {}}
MunkiImporter
{'Input': {'MUNKI_REPO': '/Users/darren/munki_repo',
           'pkg_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2.dmg',
           'pkginfo': {'catalogs': ['testing'],
                       'description': "TextMate brings Apple's approach to "
                                      'operating systems into the world of '
                                      'text editors. By bridging UNIX '
                                      'underpinnings and GUI, TextMate '
                                      'cherry-picks the best of both worlds to '
                                      'the benefit of expert scripters and '
                                      'novice users alike.',
                       'display_name': 'TextMate 2',
                       'minimum_os_version': '10.12',
                       'name': 'TextMate2',
                       'postinstall_script': '#!/bin/sh\n'
                                             '[ -d /usr/local/bin ] || mkdir '
                                             '-p /usr/local/bin\n'
                                             '/bin/cp -f -p '
                                             '/Applications/TextMate.app/Contents/Resources/mate '
                                             '/usr/local/bin/mate\n',
                       'postuninstall_script': '#!/bin/sh\n'
                                               '[ -e /usr/local/bin/mate ] && '
                                               'rm -f /usr/local/bin/mate\n',
                       'unattended_install': True},
           'repo_subdirectory': 'apps/TextMate2'}}
MunkiImporter: Copied pkginfo to /Users/darren/munki_repo/pkgsinfo/apps/TextMate2/TextMate2-2.0.6__1.plist
MunkiImporter: Copied pkg to /Users/darren/munki_repo/pkgs/apps/TextMate2/TextMate2-2.0.6__1.dmg
{'Output': {'munki_importer_summary_result': {'data': {'catalogs': 'testing',
                                                       'name': 'TextMate2',
                                                       'pkg_repo_path': 'apps/TextMate2/TextMate2-2.0.6__1.dmg',
                                                       'pkginfo_path': 'apps/TextMate2/TextMate2-2.0.6__1.plist',
                                                       'version': '2.0.6'},
                                              'report_fields': ['name',
                                                                'version',
                                                                'catalogs',
                                                                'pkginfo_path',
                                                                'pkg_repo_path'],
                                              'summary_text': 'The following '
                                                              'new items were '
                                                              'imported into '
                                                              'Munki:'},
            'munki_info': {'_metadata': {'created_by': 'darren',
                                         'creation_date': datetime.datetime(2020, 10, 16, 15, 47, 29),
                                         'munki_version': '5.1.1.4112',
                                         'os_version': '10.15.7'},
                           'autoremove': False,
                           'catalogs': ['testing'],
                           'description': "TextMate brings Apple's approach to "
                                          'operating systems into the world of '
                                          'text editors. By bridging UNIX '
                                          'underpinnings and GUI, TextMate '
                                          'cherry-picks the best of both '
                                          'worlds to the benefit of expert '
                                          'scripters and novice users alike.',
                           'display_name': 'TextMate 2',
                           'installer_item_hash': '937abb3d30863713434663422ed6a0aee59e2b27cddffd6358f9df1564327815',
                           'installer_item_location': 'apps/TextMate2/TextMate2-2.0.6__1.dmg',
                           'installer_item_size': 18442,
                           'installer_type': 'copy_from_dmg',
                           'installs': [{'CFBundleIdentifier': 'com.macromates.TextMate',
                                         'CFBundleName': 'TextMate',
                                         'CFBundleShortVersionString': '2.0.6',
                                         'CFBundleVersion': '9800',
                                         'minosversion': '10.12',
                                         'path': '/Applications/TextMate.app',
                                         'type': 'application',
                                         'version_comparison_key': 'CFBundleShortVersionString'}],
                           'items_to_copy': [{'destination_path': '/Applications',
                                              'source_item': 'TextMate.app'}],
                           'minimum_os_version': '10.12',
                           'name': 'TextMate2',
                           'postinstall_script': '#!/bin/sh\n'
                                                 '[ -d /usr/local/bin ] || '
                                                 'mkdir -p /usr/local/bin\n'
                                                 '/bin/cp -f -p '
                                                 '/Applications/TextMate.app/Contents/Resources/mate '
                                                 '/usr/local/bin/mate\n',
                           'postuninstall_script': '#!/bin/sh\n'
                                                   '[ -e /usr/local/bin/mate ] '
                                                   '&& rm -f '
                                                   '/usr/local/bin/mate\n',
                           'unattended_install': True,
                           'uninstall_method': 'remove_copied_items',
                           'uninstallable': True,
                           'version': '2.0.6'},
            'munki_repo_changed': True,
            'pkg_repo_path': '/Users/darren/munki_repo/pkgs/apps/TextMate2/TextMate2-2.0.6__1.dmg',
            'pkginfo_repo_path': '/Users/darren/munki_repo/pkgsinfo/apps/TextMate2/TextMate2-2.0.6__1.plist'}}
{'AUTOPKG_VERSION': '2.1',
 'CHECK_FILESIZE_ONLY': False,
 'CURL_PATH': '/usr/bin/curl',
 'GIT_PATH': '/Applications/Xcode.app/Contents/Developer/usr/bin/git',
 'MUNKI_REPO': '/Users/darren/munki_repo',
 'MUNKI_REPO_SUBDIR': 'apps/TextMate2',
 'NAME': 'TextMate2',
 'PARENT_RECIPES': ['/Users/darren/Library/AutoPkg/RecipeRepos/com.github.autopkg.recipes/TextMate/TextMate2.download.recipe'],
 'PRERELEASE': '',
 'RECIPE_CACHE_DIR': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2',
 'RECIPE_DIR': '/Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate',
 'RECIPE_OVERRIDE_DIRS': ['~/Library/AutoPkg/RecipeOverrides'],
 'RECIPE_PATH': '/Users/darren/GitHub/daz_wallace/autopkg-recipes/TextMate/TextMate2.munki.recipe',
 'RECIPE_REPOS': 
 **[TRUNCATED],**
 'RECIPE_SEARCH_DIRS': 
 **[TRUNCATED],**
 'archive_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/downloads/TextMate2-2.0.6.tbz',
 'destination_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2',
 'dmg_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2.dmg',
 'dmg_root': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2',
 'download_changed': False,
 'etag': '',
 'filename': 'TextMate2-2.0.6.tbz',
 'github_repo': 'textmate/textmate',
 'include_prereleases': '',
 'input_path': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/TextMate2/TextMate.app',
 'last_modified': '',
 'munki_importer_summary_result': {'data': {'catalogs': 'testing',
                                            'name': 'TextMate2',
                                            'pkg_repo_path': 'apps/TextMate2/TextMate2-2.0.6__1.dmg',
                                            'pkginfo_path': 'apps/TextMate2/TextMate2-2.0.6__1.plist',
                                            'version': '2.0.6'},
                                   'report_fields': ['name',
                                                     'version',
                                                     'catalogs',
                                                     'pkginfo_path',
                                                     'pkg_repo_path'],
                                   'summary_text': 'The following new items '
                                                   'were imported into Munki:'},
 'munki_info': {'_metadata': {'created_by': 'darren',
                              'creation_date': datetime.datetime(2020, 10, 16, 15, 47, 29),
                              'munki_version': '5.1.1.4112',
                              'os_version': '10.15.7'},
                'autoremove': False,
                'catalogs': ['testing'],
                'description': "TextMate brings Apple's approach to operating "
                               'systems into the world of text editors. By '
                               'bridging UNIX underpinnings and GUI, TextMate '
                               'cherry-picks the best of both worlds to the '
                               'benefit of expert scripters and novice users '
                               'alike.',
                'display_name': 'TextMate 2',
                'installer_item_hash': '937abb3d30863713434663422ed6a0aee59e2b27cddffd6358f9df1564327815',
                'installer_item_location': 'apps/TextMate2/TextMate2-2.0.6__1.dmg',
                'installer_item_size': 18442,
                'installer_type': 'copy_from_dmg',
                'installs': [{'CFBundleIdentifier': 'com.macromates.TextMate',
                              'CFBundleName': 'TextMate',
                              'CFBundleShortVersionString': '2.0.6',
                              'CFBundleVersion': '9800',
                              'minosversion': '10.12',
                              'path': '/Applications/TextMate.app',
                              'type': 'application',
                              'version_comparison_key': 'CFBundleShortVersionString'}],
                'items_to_copy': [{'destination_path': '/Applications',
                                   'source_item': 'TextMate.app'}],
                'minimum_os_version': '10.12',
                'name': 'TextMate2',
                'postinstall_script': '#!/bin/sh\n'
                                      '[ -d /usr/local/bin ] || mkdir -p '
                                      '/usr/local/bin\n'
                                      '/bin/cp -f -p '
                                      '/Applications/TextMate.app/Contents/Resources/mate '
                                      '/usr/local/bin/mate\n',
                'postuninstall_script': '#!/bin/sh\n'
                                        '[ -e /usr/local/bin/mate ] && rm -f '
                                        '/usr/local/bin/mate\n',
                'unattended_install': True,
                'uninstall_method': 'remove_copied_items',
                'uninstallable': True,
                'version': '2.0.6'},
 'munki_repo_changed': True,
 'pathname': '/Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/downloads/TextMate2-2.0.6.tbz',
 'pkg_path': '/Users/darren/munki_repo/pkgs/apps/TextMate2/TextMate2-2.0.6__1.dmg',
 'pkg_repo_path': '/Users/darren/munki_repo/pkgs/apps/TextMate2/TextMate2-2.0.6__1.dmg',
 'pkginfo': {'catalogs': ['testing'],
             'description': "TextMate brings Apple's approach to operating "
                            'systems into the world of text editors. By '
                            'bridging UNIX underpinnings and GUI, TextMate '
                            'cherry-picks the best of both worlds to the '
                            'benefit of expert scripters and novice users '
                            'alike.',
             'display_name': 'TextMate 2',
             'minimum_os_version': '10.12',
             'name': 'TextMate2',
             'postinstall_script': '#!/bin/sh\n'
                                   '[ -d /usr/local/bin ] || mkdir -p '
                                   '/usr/local/bin\n'
                                   '/bin/cp -f -p '
                                   '/Applications/TextMate.app/Contents/Resources/mate '
                                   '/usr/local/bin/mate\n',
             'postuninstall_script': '#!/bin/sh\n'
                                     '[ -e /usr/local/bin/mate ] && rm -f '
                                     '/usr/local/bin/mate\n',
             'unattended_install': True},
 'pkginfo_repo_path': '/Users/darren/munki_repo/pkgsinfo/apps/TextMate2/TextMate2-2.0.6__1.plist',
 'prefetch_filename': False,
 'purge_destination': True,
 'release_notes': '',
 'repo_subdirectory': 'apps/TextMate2',
 'requirement': 'identifier "com.macromates.TextMate" and anchor apple generic '
                'and certificate 1[field.1.2.840.113635.100.6.2.6] /* exists '
                '*/ and certificate leaf[field.1.2.840.113635.100.6.1.13] /* '
                'exists */ and certificate leaf[subject.OU] = "45TL96F76G"',
 'url': 'https://github.com/textmate/textmate/releases/download/v2.0.6/TextMate_2.0.6.tbz',
 'verbose': 3,
 'version': '2.0.6'}
Receipt written to /Users/darren/Library/AutoPkg/Cache/com.github.autopkg.munki.TextMate2/receipts/TextMate2.munki-receipt-20201016-164730.plist

The following new items were imported into Munki:
    Name       Version  Catalogs  Pkginfo Path                             Pkg Repo Path                          
    ----       -------  --------  ------------                             -------------                          
    TextMate2  2.0.6    testing   apps/TextMate2/TextMate2-2.0.6.plist  apps/TextMate2/TextMate2-2.0.6.dmg  

```

**Pull request merged**